src/getRuleset.ts

```diff
@@ -0,0 +1,71 @@
+import { Optional } from '@stoplight/types';
```

Comment:
Copy of https://github.com/stoplightio/spectral/blob/develop/packages/cli/src/services/linter/utils/getRuleset.ts